docs/cookbook.rst

```diff
@@ -125,4 +125,55 @@ On the subscriber's customer object, use the charge method to generate a Stripe
     amount = Decimal(10.00)
     customer.charge(amount)
 
-Source code for the Customer.charge method is at https://github.com/pydanny/dj-stripe/blob/master/djstripe/models.py#L573-L596
\ No newline at end of file
+Source code for the Customer.charge method is at https://github.com/pydanny/dj-stripe/blob/master/djstripe/models.py#L573-L596
+
+REST api
+--------
+
+The subscriptions can be accessed through a REST api. Make sure you have
```

Comment:
Wow, never thought of this one... Had to google a bit, seems like it is very much about semantics. http://stackoverflow.com/questions/1568834/whats-the-difference-between-rest-restful talks about the difference but did not 100% clarify. http://salesforce.stackexchange.com/questions/8923/rest-api-vs-restful-web-services showed that at least at Salesforce they use the semantics of talking about RESTful webservices and a REST api.